ansible/TestbedProcessing.py

```diff
@@ -416,6 +418,89 @@ def makeLab(data, devices, testbed, outfile):
                                 entry += "\tansible_ssh_pass=" + ansible_ssh_pass
                             except:
                                 print("\t\t" + host + ": ansible_ssh_pass not found")
+                        try: #get hwsku
+                            hwsku = dev.get("hwsku")
+                            if hwsku is not None:
+                               entry += "\thwsku=" + hwsku
+                        except:
+                            print("\t\t" + host + ": hwsku not found")
+
+                        try: #get cardtype
+                            cardtype = dev.get("cardtype")
+                            if cardtype is not None:
+                               entry += "\tcardtype=" + cardtype
+                        except:
+                           print("\t\t" + host + ": cardtype not found")
+
+                        try: #get num_asics
+                            num_asic = dev.get("num_asic")
+                            if num_asic is not None:
+                               entry += "\tnum_asic=" + str( num_asic )
+                        except:
+                            print("\t\t" + host + " num_asic not found")
+ 
+                        if cardtype != 'supervisor':
+                           entry += "\tstart_switchid=" + str( start_switchid )
+                           if num_asic is not None:
+                              start_switchid += int( num_asic )
+                           else:
+                              start_switchid += 1
+
+                        try: #get frontend_asics
+                            frontend_asics = dev.get("frontend_asics")
+                            if frontend_asics is not None:
+                               entry += "\tfrontend_asics=" + frontend_asics.__str__()
+                        except:
+                            print("\t\t" + host + ": frontend_asics not found")
+
+                        try: #get asics_host_ip
+                            asics_host_ip = dev.get("asics_host_ip")
+                            if asics_host_ip is not None:
+                               entry += " \tasics_host_ip=" + str( asics_host_ip )
+                        except:
+                            print("\t\t" + host + " asics_host_ip not found")
+
+                        try: #get asics_host_ipv6
+                            asics_host_ipv6 = dev.get("asics_host_ipv6")
+                            if asics_host_ipv6 is not None:
+                               entry += "\tasics_host_ipv6=" + str( asics_host_ipv6 )
+                        except:
+                            print("\t\t" + host + " asics_host_ipv6 not found")
+
+                        try: #get voq_inband_ip
+                            voq_inband_ip = dev.get("voq_inband_ip")
+                            if voq_inband_ip is not None:
+                               entry += "\tvoq_inband_ip=" + str( voq_inband_ip )
+                        except:
```

Comment:
Need to also support voq_inband_ipv6 interface

Reply:
Sure